services/headless-lms/models/src/exercise_task_submissions.rs

```diff
@@ -507,7 +507,8 @@ pub async fn get_user_exersice_task_submissions_by_course_module_and_exercise_ty
     let res: Vec<CustomViewExerciseTaskSubmission> = sqlx::query_as!(
```

Comment:
rename the function to get_user_latest_exercise_task_submissions_by_course_module_and_exercise_type(